### PR TITLE
Add Shopware EDB section and install guide

### PR DESCRIPTION
**Pull request by tiagojsag**
This PR adds the requirements + install instructions for the EDB. As we are currently still defining the final install process, the installation instructions are intentionally empty. I will keep this PR updated as we build the final workflow

### CODE REVIEW

File: source/enterprise/tech-guide/installation-guide.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+### Using the installation package
+
+For evaluation, testing or production purposes, it's recommended using one of the available installation packages:
+- Download the installation package
+- Extract the archive to your web server's content folder
+- Enable web server access to _APPLICATION_/web
+- Execute `php setup.phar` through the command line
+- Follow the instructions
+- Copy _APPLICATION_/supervisord/edb.conf.dist to _APPLICATION_/supervisord/edb.conf and configure your `supervisor` to use it.
```
Comment (JanPietrzyk) [inline source/enterprise/tech-guide/installation-guide.md]: link the *.dist file to the supervisor configuration directory - like sites-enabled vs sites-available

File: source/enterprise/tech-guide/installation-guide.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+
+### Using the development version
+
+If you are installing the EDB in a development environment and have been granted access to the git repository for the project, you 
+should refer to the included README file for instructions on how to best set up the EDB for development purposes.
```
Comment (JanPietrzyk) [inline source/enterprise/tech-guide/installation-guide.md]: Add the requirements for shop servers here:

* shopware requirements
* ioncube - if edb is configured with a shopwareId
* deployment user: edb-deploy
* python mysql
* tar
* filessystem acl
* /tmp access
* a configured host

Comment (tiagojsag) [inline source/enterprise/tech-guide/installation-guide.md]: fixed

File: source/enterprise/tech-guide/installation-guide.md (hunk 1 above)
Comment (tiagojsag) [inline source/enterprise/tech-guide/installation-guide.md]: fixed

**Comment by JanPietrzyk** [comment]
Should this also contain a preference towards https ?

**Pull request merged**